fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -5,6 +5,8 @@
   shared_key "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
   secure yes
   ca_cert_path /etc/fluent/muxkeys/ca
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
```

Comment:
Does this need to go into ansible: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging/files

Same question regarding all the config.d files